src/ol/style/textstyle.js

```diff
@@ -77,6 +81,16 @@ ol.style.Text = function(opt_options) {
 
 
 /**
+ * The default fill color to use if no fill was set at construction time.
+ *
+ * @type {string} The color of the default fill, `#333`.
+ * @const
+ * @private
```

Comment:
Not directly related to your PR but I've been wondering if `@type` is necessary when `@const` is used. https://github.com/google/closure-compiler/wiki/A-word-about-the-type-Object is not super clear to me, but it's made me wonder. I'll ask the question on the Closure Compiler mailing list.